src/utils/statuses/getStatusBadgeCopy.ts

```diff
@@ -1,7 +1,7 @@
 // &#8209; is a non-breaking hyphen used so that statuses with a hyphen do not word break when the page width shrinks
 export const getStatusBadgeCopy = (status: string) =>
```

Comment:
Can you add a test for the case where `status` is `undefined` in the corresponding test file?